Snake/Snake/Style.cs

```diff
@@ -55,62 +44,18 @@ public string StyleMazeElement(int mazeElement)
                     return "▓";
 
                 case 4:
-                    return "■";
+                    return "☻";
```

Comment:
look at this property. may help https://msdn.microsoft.com/en-us/library/system.console.outputencoding(v=vs.110).aspx